src/pages/our_ambassadors.astro

```diff
@@ -41,6 +41,22 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
         Alex Peltzer...
       </AmbassadorCard>
 
+      <AmbassadorCard
+        name="Alex Valcourt Caron"
+        img="alexvc.jpeg"
+        country="ca"
+        github="AlexVCaron"
+        linkedin="alex-valcourt-caron-545bb72b7"
+        twitter="AlexVCaron"
+      >
+        Alex Valcourt Caron is a PhD Candidate at the <a
+          href="https://scil.usherbrooke.ca"
+          target="_blank">Sherbrooke Connectivity Imaging Laboratory (SCIL)</a>, based on the main 
```

Comment:
Hi Alex.

Could you please shorten your bio? Otherwise, it pushes the social media icons away.
You can check the preview of your PR [here](https://deploy-preview-234--nextflow-staging.netlify.app/our_ambassadors.html).

Reply:
Yes, in the works ! :wink:

Reply:
It's good now :smile: